### PR TITLE
Git-link to codeblock

### PR DESCRIPTION
**Pull request by rtk-rnjn**
image_2022-01-18_123908
This is very similar to Community bot of Python

### CODE REVIEW

File: utils/bot.py [hunk 1]
```diff
@@ -25,8 +26,11 @@ def __init__(self):
             ),
             debug_guilds=config["debug_guilds"],
         )
-
+        self.http_session = ClientSession(
+            connector=TCPConnector(resolver=AsyncResolver(), family=socket.AF_INET)
+        )
         self.on_ready_fired = False
+        self._seen_messages = 0
```
Comment (Dorukyum) [inline utils/bot.py]: What's this for?

Comment (rtk-rnjn) [reply utils/bot.py]: to be honest, for nothing... or just to keep track?

Comment (Dorukyum) [reply utils/bot.py]: I think it would be the best to keep it out of this pr as it's unrelated.

Comment (rtk-rnjn) [reply utils/bot.py]: removed. well that small var could hardly make any difference

**Comment by Dorukyum** [comment]
Thanks for your contribution

**Pull request merged**